src/app/cup/cup-rating-position-in-groups/cup-rating-position-in-groups.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { CupRatingGroup } from '@models/v2/cup/cup-rating-group.model';
+import { CupRatingService } from '@app/core/services/api/v2/cup/cup-rating.service';
+import { PaginatedResponse } from '@models/paginated-response.model';
+import { CupRatingPositionSearch } from '@app/shared/models/search/cup/cup-rating-position-search.model';
+
+@Component({
+   selector: 'app-cup-rating-position-in-groups',
+   templateUrl: './cup-rating-position-in-groups.component.html',
+   styleUrls: ['./cup-rating-position-in-groups.component.scss']
+})
+export class CupRatingPositionInGroupsComponent implements OnInit {
+   public competitionId!: number;
+   public position!: number;
+   public ratingData!: CupRatingGroup[];
+   public furtheranceData!: number[];
+
+   constructor(private route: ActivatedRoute, private cupRatingService: CupRatingService) {}
```

Comment:
Let's rename `route` to `activatedRoute` to follow naming pattern as in other files, please.